pages/cnaf/beneficiaire/index.js

```diff
@@ -167,6 +167,7 @@ export default function Beneficiaire() {
                 <th rowSpan="2">Date</th>
                 <th rowSpan="2">Fichier</th>
                 { devMode && <th rowSpan="2">Taille</th>}
+                { devMode && <th rowSpan="2">Durée<br/>(en s)</th>}
```

Comment:
L'info de durée n'est pas hyper intéressante en dev. Elle le serait plus en admin (quand on récupère les infos de Matomo).

Reply:
J'ai chronométré à la main le temps de traitement lorsque je cherchais à faire le debug du bug lié à la taille des fichiers donc l'info de durée peut être intéressante en dev.
Pour l'info de durée en admin je n'arrive pas du tout à voir de où tu parles, j'avoue que je crois avoir à peu près fait le tour de l'app et je vois pas où je pourrais le mettre en plus... Quand tu as des idées de ce genre en tête n'hésite pas à être plus précis si tu peux parce que je perd pas mal de temps à essayer de deviner ce que tu veux dire 🙏

Reply:
:ok_hand: si tu vas https://betagouv.github.io/analyse-flux-insertion/cnaf/beneficiaire et que tu coches en haut à droite tu passes en mode admin. Si tu mets ton token Matomo, tu vas pouvoir afficher tous les runs récents faits par tous les usagers.

Ça serait bien (mais pas nécessaire maintenant en fonction des prios) d'afficher les tailles des fichiers et les durées de traitement à ce moment là.

Reply:
Yes c'est ce que je voulais faire, j'ai confondu avec le devmode. Le truc, c'est que je vois pas où dans le code sont les trucs qui s'affichent en mode admin. Je vois la lib, avec la méthode fetchlog, et le component, qui est la partie de "log in" qui se met en haut à droite, mais je vois pas à quel moment se "rajoutent" des informations.